raiden/tests/fixtures/variables.py

```diff
@@ -21,13 +20,13 @@
 @pytest.fixture
 def settle_timeout():
     """ NettingChannel default settle timeout. """
-    return DEFAULT_SETTLE_TIMEOUT
+    return 10
```

Comment:
We could compute the `settle_timeout` as `number_of_nodes * (reveal_timeout + 1)`, for testing this only needs to be large enough for a mediated transfer to go through all hops.